src/cli/modify/actions/displace.rs

```diff
@@ -0,0 +1,21 @@
+use gnome_randr::display_config::{
+    physical_monitor::PhysicalMonitor, ApplyConfig, monitor_models::transform::Displacement,
+};
+
+use super::{Action};
+
+pub struct DisplacementAction {
+    pub displacement: Displacement
```

Comment:
I like what you've done here with the string parsing, but there are a couple issues with it:

1. Like I said below, I'd rather keep the scale as a separate argument and make this just the `x,y`
1. The string parsing is now in the `display_config` directory, which is supposed to be a separate library. I'd rather avoid putting CLI details in the library wherever possible, which is why I had the rotation struct to begin with. This is a relatively minor transgression though so I don't mind much.

I would make a `position` struct that gets parsed to an x and y value and use that as the property on this Action.